tests/attestation_plugin/plugin/tests.c

```diff
@@ -571,6 +620,17 @@ static void _test_claims(
                                      sgx_endorsements->items[j].data,
                                      sgx_endorsements->items[j].size) == 0);
         }
+
+        if (!is_local)
+        {
+            uint8_t fmspc[6] = {0};
+
+            _find_fmspc(
```

Comment:
what happens if this fails? and it cant be found?

Reply:
I think the field fmspc is a standard one, and will need to be found. OE would return error (before this test fails), if the value is not found - 
https://github.com/openenclave/openenclave/blob/master/common/sgx/tcbinfo.c#L675